public/app/features/explore/Explore.tsx

```diff
@@ -268,7 +268,7 @@ export class Explore extends React.PureComponent<ExploreProps> {
                 return (
                   <main className="m-t-2" style={{ width }}>
                     <ErrorBoundary>
-                      {showingStartPage && <StartPage onClickExample={this.onClickExample} />}
+                      {showingStartPage && <StartPage onClickExample={this.onClickExample} mode={mode} />}
```

Comment:
The page is not shown after switching from Logs to Metrics and then back to Logs.

Reply:
Good catch! Will check this out

Reply:
I just merged my PR for clearing queries when switching between metrics and logs (#17505) so please merge with master. I think that should fix this.